applications/fishing-map/src/features/i18n/i18nNumber.tsx

```diff
@@ -1,14 +1,33 @@
 import React, { Fragment } from 'react'
 import { useTranslation } from 'react-i18next'
 import { LocaleOptions } from 'types'
+import i18n from './i18n'
 
 type I18Number = string | number
+type I18NumberOptions =
+  | LocaleOptions
```

Comment:
Not related with PR sorry, but do you think we could rename `LocaleOptions` to `Locale` ?
`LocaleOptions` makes me think this an object with options for a locale.

Reply:
Count with it!